core/src/main/java/dev/pgm/community/squads/SquadChannel.java

```diff
@@ -0,0 +1,80 @@
+package dev.pgm.community.squads;
+
+import static net.kyori.adventure.text.Component.empty;
+import static net.kyori.adventure.text.Component.text;
+import static tc.oc.pgm.util.text.TextException.exception;
+
+import java.util.Collection;
+import java.util.List;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.TextComponent;
+import net.kyori.adventure.text.format.NamedTextColor;
+import org.bukkit.command.CommandSender;
+import org.jetbrains.annotations.Nullable;
+import tc.oc.pgm.api.PGM;
+import tc.oc.pgm.api.channels.Channel;
+import tc.oc.pgm.api.match.MatchManager;
+import tc.oc.pgm.api.player.MatchPlayer;
+import tc.oc.pgm.lib.org.incendo.cloud.context.CommandContext;
+import tc.oc.pgm.util.named.NameStyle;
+
+public class SquadChannel implements Channel<Squad> {
+
+  public static final SquadChannel INSTANCE = new SquadChannel();
+
+  private static final List<String> ALIASES = List.of("sc", "pc");
```

Comment:
Would you like to add a  1 letter alias? It looks like `s` is available. Unfortunately, `/p` is already in use by `/party`, which might have been a good option otherwise. Not required, but would be nice considering all other chat channels have a 1 letter one option.

Reply:
thing is we need something the users can recognize, /s (or, squads in general) is not a user-facing name